providers/standard/src/airflow/providers/standard/triggers/external_task.py

```diff
@@ -55,7 +55,7 @@ class WorkflowTrigger(BaseTrigger):
     def __init__(
         self,
         external_dag_id: str,
-        logical_dates: list[datetime] | None = None,
+        run_ids: list[datetime] | None,
```

Comment:
```suggestion
        run_ids: list[str] | None,
```

 I think run_id is str?